projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html

```diff
@@ -137,5 +137,15 @@
         {{ 'common.remove' | cxTranslate }}
       </button>
     </div>
+    <div *ngIf="isReadOnly && item.updateable" class="cx-actions col-12">
```

Comment:
I would not tie this feature to `isReadOnly` && `item.updateable`.

Instead maybe separate input like `allowBuyAgain` would make more sense.
There might be a case that user displays cart-items and readonly and doesn't want the "buy it again" links.